test/metabase/query_processor_test.clj

```diff
@@ -91,11 +91,13 @@
    :visibility_type :normal
    :schema_name     (data/default-schema)
    :source          :fields
-   :fk_field_id     nil})
+   :fk_field_id     nil
+   :min-value       nil
+   :max-value       nil})
 
 (defn- target-field [field]
   (when (data/fks-supported?)
-    (dissoc field :target :extra_info :schema_name :source :fk_field_id)))
+    (dissoc field :target :extra_info :schema_name :source :fk_field_id :min-value :max-value)))
```

Comment:
Is there some target-field population code that is renaming the min/max keys incorrectly here?